tasks/main.yml

```diff
@@ -6,6 +6,7 @@
     - libgtk2.0-dev
     - libjasper-dev
     - libjpeg-dev
+    - liblapacke-dev
```

Comment:
I've dug a bit more into this and found [this issue](https://github.com/opencv/opencv/issues/7970) apparently fixed by [a recent PR](https://github.com/opencv/opencv/pull/7999).
It might be that building from a commit on/after that PR will remove the necessity of the package, but seeing as it's only [a small headers package](https://packages.ubuntu.com/xenial/liblapacke-dev), I don't see it as a major problem tbh.